Source/WebCore/Modules/WebGPU/Implementation/WebGPUCompositorIntegrationImpl.h

```diff
@@ -84,7 +85,7 @@ class CompositorIntegrationImpl final : public CompositorIntegration {
 #if PLATFORM(COCOA)
     Vector<MachSendRight> recreateRenderBuffers(int width, int height) override;
 
-    Vector<RetainPtr<IOSurfaceRef>> m_renderBuffers;
+    Vector<std::unique_ptr<WebCore::IOSurface>> m_renderBuffers;
```

Comment:
Why is this a unique_ptr?

Reply:
Because IOSurface::create returns a unique_ptr